pkg/kubernetes/pv.go

```diff
@@ -51,5 +53,10 @@ func VolumeToPv(volume *api.VolumeInfoResponse,
 		pv.Spec.Glusterfs.EndpointsName = endpoint
 	}
 
+	gidStr := strconv.FormatInt(int64(volume.Gid), 10)
```

Comment:
Why isn't fmt.Sprintf("%v", volume.Gid) sufficient? Is it because of the type of Gid?

Reply:
Oh, no deep thought behind it... I was mainly copying it from the kubernetes provisioner code...
Let me try to use Sprintf...